components/image/image.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="image.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    />
+    <title>avatar</title>
+  </head>
+  <body>
+    <div class="centered-row space-around-row component-border p-4">
+      <div class="image-container">
+        <img
+          src="https://picsum.photos/seed/picsum/200/200"
+          alt="responsive-image"
+          class="img-responsive"
+        />
+      </div>
+
+      <!-- round corner -->
```

Comment:
If you're commenting here to specify the type of image, you should do the same for the previous div too.

Reply:
ok I will do that.